action.yml

```diff
@@ -151,7 +147,7 @@ runs:
       uses: dsaltares/fetch-gh-release-asset@1.1.2
       with:
         repo: "gruntwork-io/pipelines-cli"
-        version: "tags/${{ inputs.pipelines_cli_version }}"
+        version: "tags/${{ env.PIPELINES_CLI_VERSION }}"
```

Comment:
Break the action when not supplied to enforce the requirement.

e.g.

```bash
if [[ -z "$PIPELINES_CLI_VERSION" ]]; then
	>&2 echo "Missing required environment variable PIPELINES_CLI_VERSION"
	exit 1
fi
```

Reply:
This already occurs in `check-msv.sh`, do we want to repeat the check?

Reply:
Ya. Now that the action requires the env var set externally, we should have visible code in the action that checks it before trying to use it in the script.

I missed that we do it there, and I think it's easy to think that the fact that the environment variable is not set before calling the script is a bug instead of intended functionality, otherwise.

Reply:
Added check using `::error::` [workflow command](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-error-message)